com.unity.uiwidgets/Runtime/cupertino/text_selection.cs

```diff
@@ -299,9 +299,10 @@ class _CupertinoTextSelectionControls : TextSelectionControls {
                     );
                 case TextSelectionHandleType.right:
                     return new Transform(
-                        transform: Matrix3.makeTrans(
+                        transform:new Matrix4().translationValues(
```

Comment:
please format this PR according to https://github.com/Unity-Technologies/com.unity.uiwidgets/blob/master/CONTRIBUTING.md

Reply:
UIWidgetCleanupPlugin.DotSettings. needs some updates.